virttest/qemu_devices/qdevices.py

```diff
@@ -460,6 +476,20 @@ def cmdline_nd(self):
             out = out[:-1]
         return out
 
+    @staticmethod
+    def _convert_raw_style_to_json_style_args(args):
```

Comment:
Looks it is unnecessary to introduce this virtual prototype to the `QCustomDevice` so I'd suggest removing it.